docker-compose.yml

```diff
@@ -296,3 +296,76 @@ services:
       - amazonaws:127.0.0.1
     depends_on:
       - dynamodb
+
+  ## AWS Plugins
+  grapl-aws-guardduty-subgraph-generator:
+    image: grapl/grapl-aws-guardduty-subgraph-generator
+    tty: false
+    environment:
+      - BUCKET_PREFIX="local-grapl"
+      - "IS_LOCAL=True"
+      - "QUEUE_URL=http://sqs.us-east-1.amazonaws.com:9324/queue/aws-guardduty-subgraph-generator-queue"
+    links:
+      - s3:minio
+      - sqs:sqs.us-east-1.amazonaws.com
+    extra_hosts:
+      - sqs:127.0.0.1
+      - amazonaws:127.0.0.1
+      - minio:127.0.0.1
+    depends_on:
+      - s3
+      - sqs
+      - grapl-aws-provision
+
+  grapl-aws-config-subgraph-generator:
+    image: grapl/grapl-aws-config-subgraph-generator
+    tty: false
+    environment:
+      - BUCKET_PREFIX="local-grapl"
+      - "IS_LOCAL=True"
+      - "QUEUE_URL=http://sqs.us-east-1.amazonaws.com:9324/queue/aws-config-subgraph-generator-queue"
```

Comment:
Remove all of the AWS references from the dockerfiles, that's just for testing the plugins.